modules/cli-options/src/main/scala/scala/cli/commands/SpecificationLevel.scala

```diff
@@ -0,0 +1,100 @@
+package scala.cli.commands
+
+import caseapp.annotation.Tag
+import os.copy
+
+sealed trait SpecificationLevel extends Product with Serializable {
+  def md = toString() + " have"
+}
+
+/** Specification levels in the context of Scala CLI runner specification. For more refer to
+  * [SIP-46](https://github.com/scala/improvement-proposals/pull/46)
```

Comment:
Since the PR has been merged, what do you think of linking to the website version? https://docs.scala-lang.org/sips/scala-cli.html